pytest_relaxed/classes.py

```diff
@@ -3,12 +3,22 @@
 
 import six
 
-from pytest import Class, Instance, Module
+from pytest import __version__ as pytest_version
+from pytest import Class, Module
 
 # NOTE: don't see any other way to get access to pytest innards besides using
 # the underscored name :(
 from _pytest.python import PyCollector
 
+pytest_version_info = tuple(map(int, pytest_version.split(".")[:3]))
```

Comment:
How about this then? It would allow pre-releases from pytest 7 onwards. The sorting of the pre-release is not relevant in the context here.

```suggestion
from pytest import Class, Module

# NOTE: don't see any other way to get access to pytest innards besides using
# the underscored name :(
from _pytest.python import PyCollector

try:
    from pytest import version_tuple as pytest_version_info
except ImportError:
    from pytest import __version__ as pytest_version
    pytest_version_info = tuple(map(int, pytest_version.split(".")[:3]))
```